src/uct/ib/mlx5/exp/ib_exp_md.c

```diff
@@ -450,6 +450,8 @@ static ucs_status_t uct_ib_mlx5_exp_reg_multithreaded(uct_ib_md_t *ibmd,
     ksm_data->mr_num = mr_num;
     status = uct_ib_md_handle_mr_list_multithreaded(ibmd, address, length,
                                                     access, chunk, ksm_data->mrs);
+    ucs_trace("MT register memory %p..%p chunks %d status %d",
```

Comment:
print this before (if memory registration fails it will have err msg anyway)
print # of threads, maybe cpu affinity, etc - maybe need to move the print inside uct_ib_md_handle_mr_list_multithreaded